pkgs/tools/misc/phoronix-test-suite/tests.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, phoronix-test-suite }:
+
+with phoronix-test-suite;
+
+stdenv.mkDerivation {
```

Comment:
Instead of `mkDerivation`? What is the difference?

can i still set a timeout? This test take 1,5 sec. max, so when it takes more than 3, someone should check out what's going on...

Here someone uses `meta` in `runCommand` env: https://github.com/NixOS/nixpkgs/blob/fb6260fcf7c126619962978b366f230cd61afe3e/pkgs/test/patch-shebangs/default.nix#L16

But it seems not to work with timeout:

```
[davidak@X230:~/code/nixpkgs/pkgs/tools/misc/phoronix-test-suite]$ time nix-build ~/code/nixpkgs -A phoronix-test-suite.tests
these derivations will be built:
  /nix/store/bh6vxi8q9lfmqlca7m9yl8yq6fjgqb0p-phoronix-test-suite-tests.drv
building '/nix/store/bh6vxi8q9lfmqlca7m9yl8yq6fjgqb0p-phoronix-test-suite-tests.drv'...
/nix/store/krqw6pmlwd12ryqvh419vm1m3a4h1npb-phoronix-test-suite-tests

real	0m6,621s
user	0m0,379s
sys	0m0,045s
```

I would expect the test to fail when it take longer than the timeout.

Reply:
I think `meta.timeout` only works on Hydra. And `runCommand` is just a wrapper over `stdenv.mkDerivation` https://github.com/NixOS/nixpkgs/blob/61f135cef67d4db42bc64c748aca1016424fa763/pkgs/build-support/trivial-builders.nix#L6